dns_utils.py

```
@@ -0,0 +1,53 @@
+import dns.resolver
+from typing import Optional
+
+resolver = dns.resolver.Resolver(configure=False)
+resolver.nameservers = ['8.8.8.8']
+resolver.timeout = 2
+resolver.lifetime = 4
+
+
+def _safe_txt_query(name: str) -> Optional[list]:
+    """Utility – return list of TXT strings or None on failure."""
```

Ruff (0.12.2)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)